manifest.json

```diff
@@ -41,8 +42,8 @@
     "https://sponsor.ajay.app/*"
   ],
   "browser_action": {
-    "default_title": "SponsorBlock",
-    "default_popup": "popup.html"
+    "default_title": "__MSG_Name__",
+    "default_popup": "__MSG_popup__"
```

Comment:
Where is this defined?

Reply:
https://github.com/OfficialNoob/SponsorBlock/blob/patch-18/_locales/en/messages.json

Reply:
Oh, does `__MSG_popup_` just refer to `popup`?

Reply:
yes